enginetest/queries/script_queries.go

```diff
@@ -6182,6 +6182,34 @@ where
 			},
 		},
 	},
+	{
+		Name: "alter statements apply correctly",
```

Comment:
Kind of a vague name, what specifically are you testing here? How is this not covered in TestAlterTable e.g.?